servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpResponseStatus.java

```diff
@@ -31,319 +31,439 @@
+     * 413 REQUEST_ENTITY_TOO_LARGE
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc7231#section-6.5.11">RFC7231, section 6.5.11</a>
      */
     public static final HttpResponseStatus REQUEST_ENTITY_TOO_LARGE = new HttpResponseStatus(413,
             "Request Entity Too Large");
 
     /**
      * 414 Request-URI Too Long
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc7231#section-6.5.12">RFC7231, section 6.5.12</a>
      */
     public static final HttpResponseStatus REQUEST_URI_TOO_LONG = new HttpResponseStatus(414, "Request-URI Too Long");
 
     /**
      * 415 Unsupported Media Type
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc7231#section-6.5.13">RFC7231, section 6.5.13</a>
      */
     public static final HttpResponseStatus UNSUPPORTED_MEDIA_TYPE = new HttpResponseStatus(415,
             "Unsupported Media Type");
 
     /**
      * 416 Requested Range Not Satisfiable
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc7233#section-4.4">RFC7233, section 4.4</a>
      */
     public static final HttpResponseStatus REQUESTED_RANGE_NOT_SATISFIABLE = new HttpResponseStatus(416,
             "Requested Range Not Satisfiable");
 
     /**
      * 417 Expectation Failed
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc7231#section-6.5.14">RFC7231, section 6.5.14</a>
      */
     public static final HttpResponseStatus EXPECTATION_FAILED = new HttpResponseStatus(417, "Expectation Failed");
 
     /**
      * 421 Misdirected Request
-     * <p>
-     * <a href="https://tools.ietf.org/html/draft-ietf-httpbis-http2-15#section-9.1.2">421 Status Code</a>
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc7540#section-9.1.2">RFC7540, section 9.1.2</a>
      */
     public static final HttpResponseStatus MISDIRECTED_REQUEST = new HttpResponseStatus(421, "Misdirected Request");
 
     /**
-     * 422 Unprocessable Entity (WebDAV, RFC4918)
+     * 422 Unprocessable Entity
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc4918#section-11.2">RFC4918, section 11.2</a>
      */
     public static final HttpResponseStatus UNPROCESSABLE_ENTITY = new HttpResponseStatus(422, "Unprocessable Entity");
 
     /**
-     * 423 Locked (WebDAV, RFC4918)
+     * 423 Locked
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc4918#section-11.3">RFC4918, section 11.3</a>
      */
     public static final HttpResponseStatus LOCKED = new HttpResponseStatus(423, "Locked");
 
     /**
-     * 424 Failed Dependency (WebDAV, RFC4918)
+     * 424 Failed Dependency
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc4918#section-11.4">RFC4918, section 11.4</a>
      */
     public static final HttpResponseStatus FAILED_DEPENDENCY = new HttpResponseStatus(424, "Failed Dependency");
 
     /**
-     * 425 Unordered Collection (WebDAV, RFC3648)
+     * 425 Unordered Collection
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc8470#section-5.2">RFC8470, section 5.2</a>
```

Comment:
Not sure that we need an incorrect link here (for `Too Early` status) until we fix the status code name, let's skip it for now.

Reply:
hm, Let me put the correct RFC draft link https://tools.ietf.org/html/draft-ietf-webdav-ordering-protocol-00#section-11.1 for Unordered Collection before it was removed.